### DISCUSSION TITLE
Some questions about the training set and the charge feature.

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/deepmd-kit#560] [Q&A]
1. If we want to use the DP to find elementary reactions (such as decomposition reaction C = A+ B), does it mean that A, B, and C molecules (and other possible substances) should be included in the training set?
2. Because the electronic term is not involved in MD (charge information is usually replaced by atomic charge), if B is a free radical or ion, can DP be effective in finding reactions involving free radicals or ions?
3. Will the DP team consider using atomic charge information as one of the input features in the future?

**Comment by njzjz** [comment]
For question 1, you may read our recent paper https://doi.org/10.1021/acs.energyfuels.0c03211, which was proposed to use DP to find a complex decomposition reaction.

**Reply by Zhang-pchao** [reply]
Thank you for responding to the question 1!

I noticed that free radicals appeared in the combustion reaction mechanism in [Nat Commun 2020, 11 (1), 5713].  For example (reaction pathway R2 in the Fig. 2), did you guess that H is a free radical instead of an ion? If in some unknown reaction, is it impossible to simply judge the state of H (this is equivalent to the question 2)?

**Reply by z5476t4508** [reply]
Hi,

Thank you for the question. In our simulation, we consider all species to be neutral. Because under combustion conditions there are few long-lifetime ions.

We are now also planning to use charge as one of the descriptors of molecules, and related work is ongoing. If you are interested, we look forward to cooperation.

**Reply by Zhang-pchao** [reply]
Thank you for your reply! My current basic knowledge is not rich enough. If there is a chance in the future, we will  be happy to cooperate!

Compared with exploring the complex reaction mechanism of small systems (AIMD may be directly used for reactions involving complex charge transfer ?), is DP more suitable for studying simple reactions of large systems (e.g. physical adsorption and desorption)?

**Reply by z5476t4508** [reply]
Yes, I think DP is more suitable for simple reactions of large system.
